src/site/content/en/blog/gedi/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: |
+  GEDI Digital reduced mobile bounce rate by 8% with Core Web Vitals
+subhead: |
+  By focusing development effort on performance improvement and applying the
+  latest web technologies and techniques, GEDI Digital has created one of the
+  fastest newspaper sites.
+authors:
+  - maurizioparadisi
+date: 2021-12-10
+hero: image/VbsHyyQopiec0718rMq2kTE1hke2/qg8rHQ5qlKK9t4Z6qTH2.jpeg
```

Comment:
Per [our media guidelines](https://web.dev/handbook/markup-media/), hero images should be 3200x960. This is too small.

Reply:
That's so large! Okay, I'm on it.